<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>376</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 251, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Aba_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>471</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="W_formulario">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Moeda de Destino:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_Dest_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Configurações do Banco de Dados</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cliente_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Configurações do Câmbio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_Janela_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Próximo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nome do cliente:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Moeda de Origem:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_Orig_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="orig_select_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selecionada:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dest_select_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selecionada:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_moeda_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Salvar as moedas "/>
      </property>
      <property name="statusTip">
       <string extracomment="Salvar as moedas "/>
      </property>
      <property name="whatsThis">
       <string extracomment="Salvar as moedas "/>
      </property>
      <property name="accessibleDescription">
       <string extracomment="Salvar as moedas "/>
      </property>
      <property name="text">
       <string>Salvar Informações</string>
      </property>
     </widget>
     <widget class="QPushButton" name="window_moeda_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Moedas</string>
      </property>
     </widget>
     <widget class="QLabel" name="tabela">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Banco de Dados:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="choose_table_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizar Tabela</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>451</width>
        <height>241</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>21</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>16</width>
        <height>81</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>451</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>20</width>
        <height>141</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_12">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>20</width>
        <height>141</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="bd1_f">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="Line" name="line_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="calen_b">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
     <zorder>line</zorder>
     <zorder>line_3</zorder>
     <zorder>line_9</zorder>
     <zorder>line_2</zorder>
     <zorder>line_11</zorder>
     <zorder>orig_select_3</zorder>
     <zorder>label_16</zorder>
     <zorder>next_Janela_3</zorder>
     <zorder>line_12</zorder>
     <zorder>line_10</zorder>
     <zorder>line_4</zorder>
     <zorder>window_moeda_3</zorder>
     <zorder>save_moeda_3</zorder>
     <zorder>dest_select_3</zorder>
     <zorder>combo_Orig_3</zorder>
     <zorder>label_12</zorder>
     <zorder>combo_Dest_3</zorder>
     <zorder>label_14</zorder>
     <zorder>cliente_3</zorder>
     <zorder>line_18</zorder>
     <zorder>label_13</zorder>
     <zorder>choose_table_3</zorder>
     <zorder>tabela</zorder>
     <zorder>label_15</zorder>
     <zorder>bd1_f</zorder>
     <zorder>calen_b</zorder>
    </widget>
    <widget class="QPushButton" name="back_janela_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QLabel" name="moeda_orig_cal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moeda de Origem:</string>
     </property>
    </widget>
    <widget class="QLabel" name="moeda_dest_cal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moeda de Destino:</string>
     </property>
    </widget>
    <widget class="QLabel" name="camb_cal">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Taxa de Câmbio:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="valor_cal">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Insira um Valor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultado_cal">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Resultado Total:</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_camb_cal">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total do Câmbio:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calcular_cal">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_all">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpar Tudo</string>
     </property>
    </widget>
    <widget class="QLabel" name="convert_calculo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Conversão Total:</string>
     </property>
    </widget>
    <widget class="Line" name="line_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_16">
     <property name="geometry">
      <rect>
       <x>-7</x>
       <y>10</y>
       <width>31</width>
       <height>291</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_17">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>31</width>
       <height>271</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>line_14</zorder>
    <zorder>line_17</zorder>
    <zorder>line_16</zorder>
    <zorder>convert_calculo</zorder>
    <zorder>line_13</zorder>
    <zorder>calcular_cal</zorder>
    <zorder>moeda_dest_cal</zorder>
    <zorder>camb_cal</zorder>
    <zorder>line_15</zorder>
    <zorder>moeda_orig_cal</zorder>
    <zorder>label_40</zorder>
    <zorder>valor_cal</zorder>
    <zorder>total_camb_cal</zorder>
    <zorder>resultado_cal</zorder>
    <zorder>clear_all</zorder>
    <zorder>back_janela_2</zorder>
    <zorder>W_formulario</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_nicio_arq">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="salve_all_3"/>
    <addaction name="create_db"/>
   </widget>
   <addaction name="menu_nicio_arq"/>
  </widget>
  <action name="salve_all_3">
   <property name="text">
    <string>Salvar Dados</string>
   </property>
  </action>
  <action name="create_db">
   <property name="text">
    <string>Criar Banco Dados</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
